snapshot-labs/snapshot
# Added Coingecko to space setting

Fixes Issue #3225

Changes in this PR:
- [x] Added Coingecko id to space setting

How to test and review this PR?
- Create new space flow should have new input field to enter Coingecko id in the settings
- Space profile page should have Coingecko logo with clickable url, if configured in the settings
- Edit space settings should have new input field to add/edit Coingecko id

### Blocker
- Cannot be tested until the Hub GraphQL schema is updated: https://github.com/snapshot-labs/snapshot-hub/pull/487
